anitya/tests/test_config.py

```diff
@@ -163,6 +165,7 @@ def test_full_config_file(self, mock_exists, mock_log):
             'SOCIAL_AUTH_LOGIN_URL': '/login/',
             'SOCIAL_AUTH_LOGIN_REDIRECT_URL': '/',
             'SOCIAL_AUTH_LOGIN_ERROR_URL': '/login-error/',
+            'CREATE_LIBRARIESIO_PROJECTS': True,
```

Comment:
Should this be `'CREATE_LIBRARIESIO_PROJECTS': create_librariesio_projects` to make sure the two test settings always stay in sync?

Reply:
I'm not entirely sure what you mean here, could you rephrase this or show me what you mean? Thanks!

Reply:
Further up in this file, you have the line `create_librariesio_projects = True`. Presumably that global and this setting are supposed to be kept consistent, so the code in my comment was meant to be a literal replacement for the current dict entry.

Alternatively, if the global isn't actually used anywhere (I didn't check), it could be deleted entirely.

Reply:
Ahhh I see. I think the diff is not very clear. That's inside a string used as the test configuration file's content.

Reply:
Ah, you're right - I only looked at the diff, not the full original file. In that case, cool, it makes sense as is :)